res/css/views/elements/_ResizeHandle.pcss

```diff
@@ -15,28 +15,29 @@ limitations under the License.
 */
 
 .mx_ResizeHandle {
+    --size: 5px;
```

Comment:
When you introduce a variable, please can you write a comment explaining what it means?

In this case: I think it is *half* the size of the displayed handle?

See also https://github.com/vector-im/element-web/blob/develop/code_style.md#comments

Reply:
Thanks for pointing out- I myself am not fully sure it would be fine to replace the values of `mx_ResizeHandle_horizontal` and `mx_ResizeHandle_vertical` by a variable. There seems to be a trick there, according to https://github.com/matrix-org/matrix-react-sdk/commit/24d9e8e16896a8054093f4f9bba11b022ec33b89, but it is not sure how those values work. Therefore, I'm reverting the commit which introduced that variable.